core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -44,6 +45,21 @@ default void onSuccess(
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node) {}
 
+  /**
+   * @deprecated This method only exists for backward compatibility. Override {@link
+   *     #onSuccess(Request, long, DriverExecutionProfile, Node, String, ExecutionInfo)} instead.
+   */
+  @Deprecated
+  default void onSuccess(
```

Comment:
This one is still being called from `GraphRequestHandler` and `ContinuousRequestHandlerBase` and some tests - should those be calling the version including the execution info so that implementers of the non-deprecated version don't miss events?

Reply:
Cleaned up all of the tests to use the new methods.